client/src/java/com/zimbra/client/ZMailbox.java

```diff
@@ -619,11 +633,13 @@ public ZMailbox(Options options) throws ServiceException {
         if (options.getTargetAccount() != null) {
             initTargetAccount(options.getTargetAccount(), options.getTargetAccountBy());
         }
+        ZimbraLog.imap.error("Created an instance of ZMailbox %s", Integer.toHexString(hashCode()), new Exception());
     }
 
     public boolean addEventHandler(ZEventHandler handler) {
         if (!mHandlers.contains(handler)) {
             mHandlers.add(handler);
+            ZimbraLog.imap.debug("Adding ZEventHandler %s. Total %d handlers. this: %s :: %s", handler.getClass().getName(), mHandlers.size(), this.toString(), Integer.toHexString(hashCode()));
```

Comment:
sorry, I should remove a bunch of debug statements that I added to figure out what was going on